examples/language-sdk-instrumentation/golang-push/rideshare/Dockerfile.load-generator

```diff
@@ -2,5 +2,5 @@ FROM golang:1.23.7
 
 WORKDIR /go/src/app
 COPY . .
-RUN go build loadgen.go
+RUN go build -tags loadgen -o loadgen .
```

Comment:
I think the way you're building the load generator is making it not to be profiled. Why do you need it this way?